telehealth-example/src/main/java/com/pubnub/components/example/telehealth/RepositoryProvider.kt

```diff
@@ -0,0 +1,155 @@
+import androidx.compose.ui.unit.LayoutDirection
+import com.pubnub.api.PubNub
+import com.pubnub.components.PubNubDatabase
+import com.pubnub.components.asPubNub
+import com.pubnub.components.chat.provider.*
+import com.pubnub.components.chat.ui.component.channel.DefaultChannelListTheme
+import com.pubnub.components.chat.ui.component.channel.LocalChannelListTheme
+import com.pubnub.components.chat.ui.component.input.DefaultLocalMessageInputTheme
+import com.pubnub.components.chat.ui.component.input.DefaultTypingIndicatorTheme
+import com.pubnub.components.chat.ui.component.input.LocalMessageInputTheme
+import com.pubnub.components.chat.ui.component.input.LocalTypingIndicatorTheme
+import com.pubnub.components.chat.ui.component.member.DefaultMemberListTheme
+import com.pubnub.components.chat.ui.component.member.LocalMemberListTheme
+import com.pubnub.components.chat.ui.component.member.MemberUi
+import com.pubnub.components.chat.ui.component.menu.DefaultMenuItemTheme
+import com.pubnub.components.chat.ui.component.menu.LocalMenuItemTheme
+import com.pubnub.components.chat.ui.component.message.*
+import com.pubnub.components.chat.ui.component.message.reaction.DefaultReactionTheme
+import com.pubnub.components.chat.ui.component.message.reaction.LocalReactionTheme
+import com.pubnub.components.chat.ui.component.provider.LocalChannel
+import com.pubnub.components.chat.ui.component.provider.LocalPubNub
+import com.pubnub.components.chat.ui.component.provider.LocalUser
+import com.pubnub.components.chat.ui.mapper.member.DBMemberMapper
+import com.pubnub.components.data.Database
+import com.pubnub.components.data.channel.ChannelDao
+import com.pubnub.components.data.channel.DBChannel
+import com.pubnub.components.data.channel.DBChannelWithMembers
+import com.pubnub.components.data.member.DBMember
+import com.pubnub.components.data.member.DBMemberWithChannels
+import com.pubnub.components.data.member.MemberDao
+import com.pubnub.components.data.membership.DBMembership
+import com.pubnub.components.data.membership.MembershipDao
+import com.pubnub.components.data.message.DBMessage
+import com.pubnub.components.data.message.MessageDao
+import com.pubnub.components.data.message.action.DBMessageAction
+import com.pubnub.components.data.message.action.DBMessageWithActions
+import com.pubnub.components.data.message.action.MessageActionDao
+import com.pubnub.components.example.getting_started.R
+import com.pubnub.components.repository.channel.DefaultChannelRepository
+import com.pubnub.components.repository.member.DefaultMemberRepository
+import com.pubnub.components.repository.membership.DefaultMembershipRepository
+import com.pubnub.components.repository.message.DefaultMessageRepository
+import com.pubnub.components.repository.message.action.DefaultMessageActionRepository
+import com.pubnub.framework.data.ChannelId
+import com.pubnub.framework.data.UserId
+import kotlinx.coroutines.runBlocking
+
+
+@Composable
+fun ChatProvider(
```

Comment:
We need to move it inside components, we can leave a TODO comment here

Reply:
Nice. Next time we can use `//todo` in the Android Studio, it'll add a date automatically for it

Reply:
Added todo